MCprep_addon/mcprep_ui.py

```diff
@@ -915,7 +893,7 @@ class MCPREP_PT_world_tools(bpy.types.Panel):
 	"""World settings and tools"""
 	bl_label = "World Tools"
 	bl_space_type = 'VIEW_3D'
-	bl_region_type = 'TOOLS' if not util.bv28() else 'UI'
+	bl_region_type = 'UI'	
```

Comment:
Nit on extra space at end of line. Not that there aren't more egregious code style issues pre-existing 😆

Reply:
I'm considering adding https://github.com/akaihola/darker in a later PR so that we can gradually add better formatting, so hopefully this will be resolved in the future